package.json

```diff
@@ -16,8 +16,8 @@
   "scripts": {
     "build": "tsc --project tsconfig.build.json && npm run --workspaces build",
     "build-fdb": "tsc --project tsconfig.build-fdb.json && npm run --workspaces build",
-    "lint": "eslint --ignore-pattern src/foundationdb/ --ignore-pattern tests/foundationdb/ --ignore-pattern **/coverage/ .",
-    "lint-fdb": "eslint --ignore-pattern **/coverage/ .",
+    "lint": "rm -rf packages/create/templates/hello-prisma/node_modules/ && eslint --ignore-pattern src/foundationdb/ --ignore-pattern tests/foundationdb/ --ignore-pattern **/coverage/ .",
+    "lint-fdb": "rm -rf packages/create/templates/hello-prisma/node_modules/ && eslint --ignore-pattern **/coverage/ .",
```

Comment:
Does that mean that if I do:
* Build
* Lint
* Run
Then it fails and I have to rebuild again?

Reply:
`packages/create/templates/hello-prisma/node_modules/ ` is just for tests. What do you mean rebuild?

Reply:
Ha got it, sorry I didn't pay enough attention and thought it was a generated prisma client.